UpdateAndStart.sh

```diff
@@ -0,0 +1,7 @@
+git --version
+git pull --rebase --autostash
+echo "Installing Node Modules..."
+export NODE_ENV=production
+npm install --no-audit --no-fund --loglevel=error --no-progress --omit=dev
```

Comment:
Users who have alternatives installed are unlikely to prefer `npm`.
```suggestion
PM=$(command -v bun || command -v yarn || command -v npm)
$PM install --no-audit --no-fund --loglevel=error --no-progress --omit=dev
```